config/experiments/ccube/ccube_pigeon_new.yaml

```diff
@@ -0,0 +1,76 @@
+# @package _global_
+# Like hawk but with logit for EOS from source
```

Comment:
I think we are missing parts of the zoo.

Reply:
https://github.com/alexhernandezgarcia/gflownet/pull/214/commits/7ba15e4d3d3af3f9257effaaa232eb746623c67a